src/Tooltip/Tooltip.component.tsx

```diff
@@ -0,0 +1,262 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled, { css } from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+
+export type Position =
+    | 'topStart'
+    | 'top'
+    | 'topEnd'
+    | 'rightStart'
+    | 'right'
+    | 'rightEnd'
+    | 'bottomEnd'
+    | 'bottom'
+    | 'bottomStart'
+    | 'leftEnd'
+    | 'left'
+    | 'leftStart';
+
+interface TooltipContainerProps extends React.HTMLAttributes<HTMLDivElement> {
+    content: string;
+    position?: Position;
+    maxWidth?: string;
+    background?: string;
+    color?: string;
+    wrap?: boolean;
+}
+
+interface TooltipContainerState {
+    width: number;
+    height: number;
+    toolTipHeight: number;
+    toolTipWidth: number;
+    hidden: boolean;
+}
+
+const ToolTipContainer = styled('div')<TooltipContainerProps>`
+    position: relative;
+    display: inline-block;
```

Comment:
We may want to make this a prop, because its a wrapper over their content. They may want it to play differently in the DOM than inline-block at times.

Reply:
👍